src/com/haskforce/settings/HaskellToolsConfigurable.java

```diff
@@ -194,7 +194,7 @@ public void updateVersion() {
             if (pathText.isEmpty()) {
                 versionField.setText("");
             } else {
-                versionField.setText(getVersion(pathText, versionParam));
+                versionField.setText(getVersion(pathText, versionParam).split("\r\n|\r|\n")[0]);
```

Comment:
Note that this is unsafe for two reasons: `getVersion` might return `null` and `split` might return an empty array.

I've appended a commit to resolve these issues - 0c3fca3